healthcare_facility_of_origin_city.py

```diff
@@ -0,0 +1,55 @@
+import pandas as pd  # data manipulation
+import logging  # log errors and information
+import sys  # system-specific parameters and functions
+
+"""
+Author: Tom Iverson
+Released: 2023-11-30
+Version: 1.0.0
+Description:
+This script's main functionality is to determine a samples' city from the C. auris LIMS export data (ARLN_Specimen_ID, Healthcare_facility_of_origin_name, and Healthcare_facility_of_origin_state).
+It does this by taking a 'samples.txt' file that the user has created with the following headers "ARLN_Specimen_ID, Healthcare_facility_of_origin_name, Healthcare_facility_of_origin_state" and merges
+the data with 'C_auris_Healthcare_Facility_of_origin_name_city_state.txt'.
```

Comment:
C_auris_Healthcare_Facility_of_origin_name_city_state.txt is this a file that you manually maintain? How do you find the locations of the facilities? Just through google?

Reply:
Clarified the readme to answer these questions.